### PR TITLE
Created animals table

### PR DESCRIPTION
**Pull request by CobbyElsonfx**
## In this PR , I implemented the following :
- Created a table called animals using PostgreSQL
- Added the following columns to the table:
      id: integer
      name: string
      date_of_birth: date
      escape_attempts: integer
      neutered: boolean
      weight_kg: decimal

### Click on the link to access  Project Requirements
 [Project Requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-)
Query 1
- Find all animals whose name ends in "mon".

structure/vet_clinic_create_tables.md)

Query 2.
- List the name of all animals born between 2016 and 2019.
second
 
Query 3
- List the name of all animals that are neutered and have less than 3 escape attempts.
thired

Query 4
- List the date of birth of all animals named either "Agumon" or "Pikachu".
fourth

Query 5
- List name and escape attempts of animals that weigh more than 10.5kg
fifth

Query6
- Find all animals that are neutered.
sixth

Query 7
- Find all animals not named Gabumon.
seventh

Query 8
Find all animals with a weight between 10.4kg and 17.3kg
-  (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
eight

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +171,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (zeff96) [inline README.md:172]: - [ ] Hello, kindly have a look at this link, there is a spelling mismatch and as a result, I get the below error. Kindly correct spelling on either file name or link name as you please and make sure both file name and relative link path matches

Screenshot from 2023-10-12 09-04-44

**Review by zeff96 — CHANGES_REQUESTED** [review]
### Status: CHANGES REQUIRED ♻️ ♻️ ♻️ 
Hi @CobbyElsonfx , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close

## Highlights

:heavy_check_mark: Correct workflow
:heavy_check_mark: Descriptive pull request

## Required Changes ♻️

- [ ] 👀 Hey there! Would you mind taking a look at the first screenshot and helping me make it better? 📸 I'd like to ensure that it can display effectively. 🛠️💡

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (CobbyElsonfx) [reply README.md:172]: Thanks ,I will resolve the problem

**Review by divinecharlotte — APPROVED** [review]
# Hello @CobbyElsonfx  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- Good implementation of the required changes✔️
- Descriptive pr✔️






## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**